Source/Renderer/Context.js

```diff
@@ -523,7 +525,7 @@ define([
          */
         floatingPointTexture : {
             get : function() {
-                return this._textureFloat;
+                return this._textureFloat || this._webgl2;
```

Comment:
This might've gotten lost in one of the other closed PRs, but this isn't true. To render to a floating point texture in WebGL 1, you need support for the `OES_texture_float` extension. To render to a floating point texture in WebGL 2, you need support for `EXT_color_buffer_float` extension.